src/note_encryption.rs

```diff
@@ -92,6 +92,11 @@ impl OrchardDomain {
             rho: *act.nullifier(),
         }
     }
+
+    /// Constructs a domain from a nullifier.
+    pub fn for_nullifier(nullifier: Nullifier) -> Self {
```

Comment:
For future reference, these additions should be added to the changelog. In this case, I can do this in the release PR since we'll be cutting it imminently and won't forget.